gms/gossiper.hh

```diff
@@ -104,11 +104,11 @@ private:
 
     void init_messaging_service_handler();
     future<> uninit_messaging_service_handler();
-    future<> handle_syn_msg(msg_addr from, gossip_digest_syn syn_msg);
-    future<> handle_ack_msg(msg_addr from, gossip_digest_ack ack_msg);
-    future<> handle_ack2_msg(msg_addr from, gossip_digest_ack2 msg);
-    future<> handle_echo_msg(msg_addr from, std::optional<int64_t> generation_number_opt);
-    future<> handle_shutdown_msg(msg_addr from, std::optional<int64_t> generation_number_opt);
+    future<> handle_syn_msg(const rpc::client_info& cinfo, gossip_digest_syn syn_msg);
+    future<> handle_ack_msg(const rpc::client_info& cinfo, gossip_digest_ack ack_msg);
+    future<> handle_ack2_msg(const rpc::client_info& cinfo, gossip_digest_ack2 msg);
+    future<> handle_echo_msg(const rpc::client_info& cinfo, std::optional<int64_t> generation_number_opt);
+    future<> handle_shutdown_msg(const rpc::client_info& cinfo, std::optional<int64_t> generation_number_opt);
```

Comment:
Isn't it time to move gossiper verbs to the IDL?

Reply:
Yeah, it could be done as a preliminary cleanup PR